core/model/src/main/java/org/eclipse/rdf4j/model/util/Literals.java

```diff
@@ -580,6 +582,22 @@ public static boolean isLanguageLiteral(Literal literal) {
 		return Objects.requireNonNull(literal, "Literal cannot be null").getLanguage().isPresent();
 	}
 
+	/**
+	 * Normalises the given <a href="https://tools.ietf.org/html/bcp47">BCP47</a> language tag according to
+	 * the rules defined by the JDK in the {@link Locale} API.
+	 * 
+	 * @param languageTag
+	 *        An unnormalised, valid, language tag
+	 * @return A normalised version of the given language tag
+	 * @throws IllformedLocaleException
+	 *         If the given language tag is illformed according to the rules specified in BCP47.
+	 */
+	public static String normaliseBCP47Tag(String languageTag)
```

Comment:
Given that BCP47 is the "format to follow" according to the RDF 1.1 spec, could we simplify the method name to just `normaliseTag`?